packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -1,170 +1,98 @@
-  width: 100%;
-  height: 100%;
 
-  content: '';
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' id='texture' data-v-1d260e0e=''%3E%3Cfilter id='noise' data-v-1d260e0e=''%3E%3CfeTurbulence type='fractalNoise' baseFrequency='.8' numOctaves='4' stitchTiles='stitch' data-v-1d260e0e=''%3E%3C/feTurbulence%3E%3CfeColorMatrix type='saturate' values='0' data-v-1d260e0e=''%3E%3C/feColorMatrix%3E%3C/filter%3E%3Crect width='100%25' height='100%25' filter='url(%23noise)' data-v-1d260e0e=''%3E%3C/rect%3E%3C/svg%3E");
-  opacity: 0.35;
-  filter: 'contrast(120%) brightness(120%)';
-}
-
-[data-amplify-authenticator] [data-amplify-container] {
-  /* Fix z-index of texture being over the Authenticator */
-  position: relative;
-  place-self: center;
-  width: 30rem;
-  @media (max-width: 30rem) {
-    width: 100%;
+  & [data-amplify-router] {
+    background-color: var(--amplify-colors-background-primary);
+    box-shadow: var(--amplify-shadows-medium);
+    border-color: var(--amplify-components-authenticator-router-border-color);
+    border-width: var(--amplify-components-authenticator-router-border-width);
+    border-style: solid;
   }
-}
-
-// footers inside each screen should have some margin below it
-[data-amplify-authenticator] [data-amplify-router] [data-amplify-footer] {
-  padding-bottom: var(--amplify-space-medium);
-  text-align: center;
-}
-
-[data-amplify-authenticator] [data-amplify-router] {
-  background-color: var(--amplify-colors-background-primary);
-  box-shadow: var(--amplify-shadows-medium);
-}
 
-[data-amplify-authenticator] [data-amplify-form] {
-  padding: var(--amplify-space-xl);
-}
-
-[data-amplify-authenticator] [data-state='inactive'][role='tab'] {
-  background-color: var(--amplify-colors-background-secondary);
-}
-
-/* Remove double-border between the password field & reveal button */
-[data-amplify-authenticator]
-  .amplify-passwordfield
-  .amplify-button:not(:focus) {
-  border-left: none;
-}
-
-[data-amplify-authenticator]
-  .amplify-phonenumberfield
-  .amplify-countrycodeselect {
-  height: 100%;
-}
+  & [data-amplify-footer] {
+    padding-bottom: var(--amplify-space-medium);
+    text-align: center;
+  }
 
-[data-amplify-authenticator]
-  .amplify-phonenumberfield
-  .amplify-select__wrapper {
-  height: 100%;
-}
+  & [data-amplify-form] {
+    padding: var(--amplify-space-xl);
+  }
 
-/* Remove double-border between the phone field & country select  */
-[data-amplify-authenticator] .amplify-phonenumberfield select:not(:focus) {
-  border-right: none;
-}
+  & [data-state='inactive'] {
+    background-color: var(--amplify-colors-background-secondary);
+  }
 
-[data-amplify-authenticator] [data-or-container] {
-  position: relative;
-  color: gray;
-  align-items: center;
-  justify-content: center;
-  padding: 1.5rem 0px;
+  & .amplify-passwordfield .amplify-button:not(:focus) {
+    border-left: none; /* Remove double-border between the password field & reveal button */
```

Comment:
I thought it was a good idea, if you meant to remove `:not(:focus)`, until I tried it....

https://user-images.githubusercontent.com/11983489/155630928-8b39f8dd-bbb4-45cf-a4c4-cadc2929084f.mov

Reply:
Could we make the z-index of the button higher when it is focused? That could work?

Reply:
No, it's not about the overlay. The thing is there are 2 borders and we don't want to see both of them at the same time, so we want to do:

1. no focus: hide the left one (only see the right one)
2. focus: show the left one (no right one either)

Reply:
What if... we didn't hide the left border of the button and instead gave it a negative margin? Then also we set things that are focused to a z-index of 2 so when either one is focused it will go above the other

![CleanShot 2022-02-25 at 14 53 58](https://user-images.githubusercontent.com/321279/155813882-f9be2180-23a7-4a88-ad7c-6dfe7fd75dad.gif)

Reply:
It's good if we can avoid using `not focus`, but I'm not sure if it's better to use `z-index`, because when I was refactoring the modal's css, I noticed that we've already used `z-index` in different numbers. 

So, even though the modal is set to be `z-index: 1`, it doesn't pop-up and we have to set it to a bigger number, which is the the problem of using `z-index`. 

I would prefer to save `z-index` for the places we have to use them. 

What do you think?

Reply:
I moved `.amplify-passwordfield .amplify-button:not(:focus)` style to `passwordFieldStyels.css` so that it applies to all the password fields